src/crud/peer.py

```
@@ -166,6 +193,12 @@ async def update_peer(
         honcho_peer.configuration = peer.configuration
 
     await db.commit()
+    await db.refresh(honcho_peer)
+
+    # Invalidate cache - read-through pattern
+    cache_key = peer_cache_key(workspace_name, honcho_peer.name)
+    await cache.delete(cache_key)
+
     logger.debug(f"Peer {peer_name} updated successfully")
```

Ruff (0.14.1)

Logging statement uses f-string

(G004)